crates/telio-firewall/src/firewall.rs

```
@@ -801,8 +840,12 @@ impl Firewall for StatefullFirewall {
     fn reset_connections(&self, pubkey: &PublicKey, sink: &mut dyn io::Write) -> io::Result<()> {
         telio_log_debug!("Constructing connetion reset packets");
 
-        self.conntracker.reset_tcp_conns(pubkey, sink)?;
-        self.conntracker.reset_udp_conns(pubkey, sink)?;
+        self.conntracker.reset_tcp_conns(pubkey, |packet: &[u8]| {
+            let _ = sink.write_all(packet);
```

This is ignoring errors that previously had been handled.